packages/formspree-core/src/core.ts

```diff
@@ -46,22 +43,17 @@ export class Client {
    * @param data - An object or FormData instance containing submission data.
    * @param args - An object of form submission data.
    */
-  async submitForm(
+  async submitForm<T extends FieldValues>(
     formKey: string,
-    data: SubmissionData,
+    data: SubmissionData<T>,
     opts: SubmissionOptions = {}
-  ): Promise<SubmissionResponse> {
+  ): Promise<SubmissionResult<T>> {
     const endpoint = opts.endpoint || 'https://formspree.io';
```

Comment:
@colevscode do you know the motivation for this to be configurable?

Reply:
testing on localhost

Reply:
Ah, that makes sense. Thanks!